File: app/src/main/java/me/wcy/music/utils/ParseUtils.java
package me.wcy.music.utils;

/**
 * Created by wcy on 2017/7/8.
 */
public class ParseUtils {

    public static long parseInt(String s) {
        try {
            return Integer.parseInt(s);
        } catch (NumberFormatException e) {
            return 0;
        }
    }

    public static long parseLong(String s) {
        try {
            return Long.parseLong(s);
        } catch (NumberFormatException e) {
            return 0;
        }
    }
}

File: app/src/main/java/me/wcy/music/utils/ParseUtils.kt
package me.wcy.music.utils

/**
 * Created by wcy on 2017/7/8.
 */
object ParseUtils {
    fun parseInt(s: String): Long {
        return try {
            s.toInt().toLong()
        } catch (e: NumberFormatException) {
            0
        }
    }

    fun parseLong(s: String?): Long {
        return try {
            s!!.toLong()
        } catch (e: NumberFormatException) {
            0
        }
    }
}